### DISCUSSION TITLE
Title Menus?

### DISCUSSION THREAD

**Discussion by winkmichael** [spectreconsole/spectre.console#1140] [Q&A] [answered]
Hello all,

I've been using Spectre of and on for a few years with great success; however, I only use it for the AnsiConsole and its lovely decoration and markup. Thanks for this great project.

In the past, I've used Terminal.Gui to create a menu-driven console program, but it's old and written in .NET Framework, and it's time to rewrite it under .NET Core. That said, I figured Spectre might be the better option, but I can't find a menu / top menu option documented anywhere.

Attached is a screenshot of what I currently have implemented in Terminal.Gui can this do in Spectre?

Thanks again
Mike
<img width="314" alt="image" src="https://user-images.githubusercontent.com/5185889/214201037-d781c1e2-af51-4b63-8b33-1a5d2163f413.png">

**Comment by patriksvensson** [comment, marked as answer]
@winkmichael Thanks for your kind words

Spectre.Console is a bad choice for this kind of app since it's not a TUI framework like [Terminal.Gui](https://github.com/gui-cs/Terminal.Gui). We have no plans of supporting scenarios like the one you describe. We recommend that you go with `Terminal.Gui` for this.

`Terminal.Gui` is actively maintained and was updated as late as 22 days ago. It also runs on `netstandard2.0` and `net6.0`: https://www.nuget.org/packages/Terminal.Gui/1.9.0#dependencies-body-tab
